### PR TITLE
Merchadise-service_CU-m18mhv

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-m18mhv Merchadise service](https://app.clickup.com/t/m18mhv)

**Comment by lpboucher** [comment]
Task linked: [CU-m18mhv Merchadise service](https://app.clickup.com/t/m18mhv)

File: src/app/core/products/merchandise-api.service.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            description: "Premium quality: 300 gr./m2",
+            slug: "tote",
+            dimensions: "41 x 42 cm",
+            material: "Cotton",
+        })
+    ];
+
+    constructor() {}
+
+    getMerchandise(): Observable<Merchandise[]> {
```
Comment (lpboucher) [inline src/app/core/products/merchandise-api.service.ts:26]: best practice (as far as I have seen it so far...) is to use getAll{resource} for the GET / route and to use getOne{resource} for the get /:id or /:slug

Just a quick note, we will fix when we change to the actual api so need to fix in this ticket

File: src/app/core/products/product-api.service.ts [hunk 2]
```diff
@@ -1,45 +1,37 @@
...
     ) {
         super(config.backendURL + "wholesale/", http, Coffee);
     }
 
     getProducts(): Observable<Product[]> {
-        return (this.coffeeApiService.getCoffees() as Observable<Product[]>)
-            .pipe(
-                map(arr => { arr.push(...this.products); return arr; })
-            );
+        return forkJoin([
```
Comment (lpboucher) [inline src/app/core/products/product-api.service.ts:29]: well done, rxjs to the rescue!

**Review by lpboucher — APPROVED** [review]

**Pull request merged**